pkgs/applications/networking/deck/default.nix

```diff
@@ -3,13 +3,13 @@ let
   short_hash = "63b9742";
```

Comment:
short_hash has to be updated manually

Reply:
Hi @NickCao 

I created this https://github.com/NixOS/nixpkgs/pull/248533 to automate this process.

Can you please have a look if that one can be merged?

Reply:
I'm not exactly happy with the complexity of the update script. The short_hash has nothing to do with the functionalities of the software, and is barely informational (as the version/tag already maps to the specified commit), I would rather have it dropped. If you insist on keeping it however, you may use `nix-update` to handle half of the work (including updating vendorHash).

Reply:
In the meantime, `short_hash` can be retrieved and stored in `fetchFromGitHub`, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/fastly/default.nix.